collectors/python.d.plugin/varnish/varnish.chart.py

```diff
@@ -172,6 +172,21 @@ def storage_charts_template(name):
 
     return order, charts
 
+def storage_alloc_objs_charts_template(name):
+    order = [
```

Comment:
How about to add this chart to the `storage_charts_template` so it returns both?

Reply:
Yes, I could also do that. Do you prefer in that way?

Reply:
Done